src/components/SpaceSidebarFooter.vue

```diff
@@ -46,6 +53,7 @@ const socials = computed<SocialItem[]>(() => {
       <i-s-twitter v-if="social.icon === 'twitter'" class="text-[23px]" />
       <i-s-github v-if="social.icon === 'github'" />
       <i-ho-globe-alt v-if="social.icon === 'earth'" class="text-[23px]" />
+      <i-s-coingecko v-if="social.icon === 'coingecko'" />
```

Comment:
Instead of changing the BaseIcon code we should replace the use of BaseIcon inside InputSocial with `<i-s-xxx />` icons like we do here in the footer

Reply:
Thanks for the feedback @samuveth. Updated the PR!